ember_debug/view-debug.js

```diff
@@ -614,10 +611,14 @@ var ViewDebug = Ember.Object.extend(PortMixin, {
    * @return {Boolean}
    */
   _isGlimmer: function() {
-    var id = Ember.keys(Ember.View.views)[0];
-    return id && !Ember.View.views[id].get('_childViews');
+    var id = Ember.keys(this.get('viewRegistry'))[0];
+    return id && !this.get('viewRegistry')[id].get('_childViews');
   },
 
+  viewRegistry: computed('application', function() {
+    return Ember.View.views || this.get('application.__container__').lookup('-view-registry:main');
```

Comment:
I think the order here should be reversed. In 1.13 Ember.View.views should be present (for legacy support).

Can you confirm (I'm on mobile ATM)? Also, if Ember.View.views (but we should still be using -view-registry if present) is not present on 1.13 I think that is also a bug...